File: exemplo1.js

    function search(arr , n , x) {
      var i;
      for (i = 0; i < n; i++) {
          if (arr[i] == x) {
              return i;
          }
      }
      return -1;
  }


      var arr = [ 1, 10, 30, 15 ];
      var x = 30;
      var n = arr.length;
      document.write(x+" is present at index "+ search(arr, n, x));

File: exemplo1.py
def sequencialSearch(arr, x):
    for index, value in enumerate(arr):
        if value == x:
            return index
    return -1
 

if __name__ == '__main__':
    arr = [10, 20, 30, 40]
    x = 30
 
    print(x, "está presente no index",
          sequencialSearch(arr, x))